Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+	* [Software Known to Detect](#software-known-to-detect) ([Alt.](#Software%20Known%20to%20Detect))
+	* [Preservation Instructions](#preservation-instructions) ([Alt.](#Preservation%20Instructions))
+	* [Associated File Attributes](#associated-file-attributes) ([Alt.](#Associated%20File%20Attributes))
+	* [Known Manufacturing Information](#known-manufacturing-information) ([Alt.](#Known%20Manufacturing%20Information))
+		* [Known Manufacturing Plants](#known-manufacturing-plants) ([Alt.](#Known%20Manufacturing%20Plants))
+		* [Known Ringcode Information](#known-ringcodeinformation) ([Alt.](#Known%20Ringcode%20Information))
+			* [Mastering Code](#mastering-code)  ([Alt.](#Mastering%20Code))
+			* [Mastering SID Code](#mastering-sid-code)  ([Alt.](#Mastering%20SID%20Code))
+			* [Toolstamp or Mastering Code](#toolstamp-or-mastering-code)  ([Alt.](#Toolstamp%20or%20Mastering%20Code))
+			* [Mould SID Code](#mould-sid-code)  ([Alt.](#Mould%20SID%20Code))
+* [Reference Material](#reference-material) ([Alt.](#Reference%20Material))
+	* [Media](#media)
+	* [Additional Resources](#additional-resources) ([Alt.](#Additional%20Resources))
+	* [Footnotes](#footnotes)
+
+*** 
+
+## Categories: 
+
+* Optical Copy Protection
+
+***
+
+## Protection Features:
+
+* Ring protection
+* Dummy files
+* Disc check (Professional)
+* SoftICE Debugger Protection (Professional)
+
+***
+
+## DRML Entry Version: 
+
+***
+
+# DRM Information:
+
+## Names: 
+
+* copy-X[^CopyX_Original_Website]
+* copy-X - the ultimate Copy-Killer.[^CopyX_Original_Website]
+* copy-X audio[^CopyX_Original_Website]
+* copy-X light[^CopyX_Original_Website]
+* copy-X professional[^CopyX_Original_Website]
+* copy-X trial maker[^CopyX_Original_Website]
+
+[^CopyX_Original_Website]: [The original optimal media link for copy-X.](https://web.archive.org/web/20011016234742/http://www.optimal-online.de:80/product/copy_x.htm)
+
+***
+
+## Developers:
+
+* optimal media production (Until 2011)[^CopyX_Current_Website]
+* optimal media (Current)
+
+[^CopyX_Current_Website]: [The current optimal media website link.](https://www.optimal-media.com/)
+***
+
+## Overall Description:
+
+Ring protection copy protection scheme from german company optimal media[^CopyX_Original_Website]. As seen in Names, four different types seem to have been in use[^CopyX_Original_Website]. Note that it has not yet been confirmed whether light and professional correspond to the currently established distinct behaviors, but it seems to be the most likely conclusion.
+Had to be mastered at a specific optimal media production plant[^CopyX_2009_Document].
+
+[^CopyX_2009_Document]: [Optimal media document from 2009, seemingly for prospective customers.](http://press9.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf)
+***
+
+## Timeline: 
+
+#### 2000s
+
+* 2001?: Seems to come into existence about here.
+
+* 2009?: Some sort of overhaul occurs.
+
+***
+
+## Versions & Differences: 
```

Comment:
Please add footnotes regarding the various protection features mentioned here, feel free to use the same or similar footnotes as the earlier "Protection Features" section.

Reply:
This review comment doesn't make as much sense as I thought it did, on further consideration. Please just add a footnote explaining how you know that the dummy files are intersected by the rings specifically.

Reply:
Fixed.